app/models/spree/calculator/avalara_transaction.rb

```diff
@@ -40,7 +40,7 @@ def can_calculate_tax?(order)
       address = order.ship_address
 
       return false unless Spree::Avatax::Config.tax_calculation
-      return false if %w(address cart).include?(order.state)
+      return false unless %w(cart payment).include?(order.state)
```

Comment:
we should probably stick to consistent array definitions, everywere else we use `%w[...]`, square parenthesis

Reply:
this is not our main rails app project so i am keeping it how it is in this project/gem